data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json

```diff
@@ -57,6 +56,11 @@
     "subtype": "collection",
     "entries": [ { "item": "cudgel" } ]
   },
+  {
+    "type": "effect_type",
+    "id": "Alonso_condoms_restored",
+    "//": "Applied as a timer to make Alonso's condoms run out over time."
```

Comment:
Hm. This isn't quite how I'd pictured it, what I was imagining was essentially Alonso taking a condom item as a "payment" to initiate "comfort" actions. What's the rationale for a resetting timer? I'm not opposed to it on principle or anything.

Reply:
I did this work assuming that "condom" was a pack of condoms like the real world, but it looks like that might not exist in the game. Single serve is actually much simpler.